cmd/autoconfigbrancher/main.go

```diff
@@ -191,6 +191,12 @@ func main() {
 
 	commitIfNeeded(fmt.Sprintf("%s --from-dir ./ci-operator/config --to-dir ./ci-operator/jobs", "ci-operator-prowgen"), author)
 
+	cmd = "/usr/bin/private-prow-configs-mirror"
+	args = []string{"--release-repo-path", "."}
+	run(cmd, args...)
+
+	commitIfNeeded(fmt.Sprintf("%s --release-repo-path .", "private-prow-configs-mirror"), author)
```

Comment:
The useless `Sprintf` triggers me, but I think I'll just file a followup PR that will fix all of them.